packages/salesforcedx-utils-vscode/src/helpers/traceFlagsRemover.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Connection } from '@salesforce/core';
+
+/**
+ * TraceFlagsRemover is a singleton which deletes trace flags not created by the user.
```

Comment:
The comment is confusing. Did you mean "deletes trace flags created by the user"?

Reply:
done